### PR TITLE
docs(angular): refresh angular plugin overview docs

### PR DESCRIPTION
**Pull request by leosvelperez**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/shared/angular-plugin.md [hunk 1]
```diff
@@ -26,6 +28,7 @@ npm install -D @nrwl/angular
 
 Building Angular applications within an Nx workspace is similar to building within a vanilla Angular CLI workspace, with a few differences.
 
+- ESLint is used as the default linter, configurable in the generators section of the workspace configuration.
 - Jest is used as the default unit test runner, configurable in the generators section of the workspace configuration file.
```
Comment (Coly010) [inline docs/shared/angular-plugin.md:32]: I know this wasn't your doing, but I feel like these three bullet points should have the same wording. The second one being slightly off is strange.

File: docs/shared/angular-plugin.md [hunk 2]
```diff
@@ -65,48 +76,64 @@ myorg/
...
+- [Using NgRx](/{{version}}/angular/guides/misc-ngrx)
+- [Upgrading an AngularJS application to Angular](/{{version}}/angular/guides/misc-upgrade)
 
 ## Executors / Builders
 
-- [package](/{{framework}}/angular/package) - Bundles artifacts for a buildable library that can be distributed as an NPM package.
+- [delegate-build](/{{framework}}/angular/delegate-build) - Delegates the build to a different target while supporting incremental builds.
+- [ng-packagr-lite](/{{framework}}/angular/ng-packagr-lite) - Builds a library with support for incremental builds.
+- [package](/{{framework}}/angular/package) - Builds and packages an Angular library to be distributed as an NPM package. It supports incremental builds.
+- [webpack-browser](/{{framework}}/angular/webpack-browser) - Builds a browser application with support for incremental builds.
```
Comment (Coly010) [inline docs/shared/angular-plugin.md:113]: And custom webpack configs.

File: docs/shared/angular-plugin.md [hunk 3]
```diff
@@ -65,48 +76,64 @@ myorg/
...
-- [stories](/{{framework}}/angular/stories) - Create stories/specs for all components declared in a library
-- [storybook-configuration](/{{framework}}/angular/storybook-configuration) - Setup configuration for Storybook
-- [upgrade-module](/{{framework}}/angular/upgrade-module) - Add an upgrade module
+- [application](/{{framework}}/angular/application) - Creates an Angular application.
+- [convert-tslint-to-eslint](/{{framework}}/angular/convert-tslint-to-eslint) - Converts a project from TSLint to ESLint.
+- [downgrade-module](/{{framework}}/angular/downgrade-module) - Sets up a Downgrade Module.
+- [karma](/{{framework}}/angular/karma) - Adds Karma configuration to a workspace.
+- [karma-project](/{{framework}}/angular/karma-project) - Adds Karma configuration to a project.
+- [library](/{{framework}}/angular/library) - Creates an Angular library.
+- [move](/{{framework}}/angular/move) - Moves an Angular application or library to another folder within the workspace.
```
Comment (Coly010) [inline docs/shared/angular-plugin.md:123]: And updates the app or lib project name within the workspace configuration

File: nx-dev/nx-dev/public/documentation/latest/shared/angular-plugin.md [hunk 4]
```diff
@@ -1,5 +1,7 @@
 # Angular Plugin
```
Comment (Coly010) [inline nx-dev/nx-dev/public/documentation/latest/shared/angular-plugin.md:1]: I guess this is just a copy and paste of the other file? lol

**Review by Coly010 — APPROVED** [review]
Looks good! Couple of suggested changes to help clarify a couple of things. Other than that looks good.

File: nx-dev/nx-dev/public/documentation/latest/shared/angular-plugin.md (hunk 4 above)
Comment (leosvelperez) [reply nx-dev/nx-dev/public/documentation/latest/shared/angular-plugin.md:1]: Yeah, this is the one that will actually be displayed on nx.dev as `Latest`. I'm modifying it now so we don't have to way for a new release to have these updates published.

**Pull request merged**